src/tests/enricher/test_enrich_bag.py

```diff
@@ -25,9 +40,39 @@ def test_enrich_nummeraanduidingen(self):
         self.assertEqual(entities[0]['ligt_in_bag_woonplaats'],'3594')
         self.assertEqual(entities[1]['ligt_in_bag_woonplaats'],'1024')
 
+        self.assertEqual(entities[0]['amsterdamse_sleutel'],'sleutel')
+        self.assertEqual(entities[1]['amsterdamse_sleutel'],'sleutel2')
+
+    @mock.patch("gobimport.enricher.bag.BAGEnricher._download_amsterdam_sleutel_file", mock.MagicMock())
+    @mock.patch("gobimport.enricher.bag.BAGEnricher._get_amsterdamse_sleutel_lookup")
+    def test_enrich_openbareruimtes(self, mock_lookup):
+        mock_lookup.return_value = {
+            '1234': {
+                'amsterdamse_sleutel': 'sleutel',
+                'straatcode': 'code'
+            }
+        }
+
+        entities = [
+            {
+                'identificatie': '1234',
+                'ligt_in_bag_woonplaats': '1024;3594',
+            }
+        ]
+
+        enricher = BAGEnricher("app", "bag", "openbareruimtes")
+        for entity in entities:
+            enricher.enrich(entity)
+
+        # Check if the both sleutel and straatcode have been added
+        self.assertEqual(entities[0]['amsterdamse_sleutel'],'sleutel')
+        self.assertEqual(entities[0]['straatcode'],'code')
```

Comment:
Also only the success path is tested

Reply:
Attribute should be empty, added in the test